src/os/bluestore/FreelistManager.h

```diff
@@ -38,6 +38,10 @@ class FreelistManager {
   virtual void release(
     uint64_t offset, uint64_t length,
     KeyValueDB::Transaction txn) = 0;
+
+  virtual bool supports_parallel_transactions() {
+    return false;
+  }
```

Comment:
@liewegas . I don't know for extentfreelistmanager  why don't support parallel transaction? Hope your explain. Thanks!